packages/ptokens-assets-evm/src/ptokens-evm-provider.ts

```diff
@@ -183,17 +199,28 @@ export class pTokensEvmProvider implements pTokensAssetProvider {
     _options: MakeContractCallOptions,
     _args: T | [] = [],
   ): Promise<R> {
-    const { method, abi, contractAddress } = _options
-    return this._publicClient.readContract({
-      address: stringUtils.addHexPrefix(contractAddress),
-      abi: abi,
-      functionName: method,
-      args: _args
-    }) as Promise<R>
+    const { method, abi, contractAddress, onInterim } = _options
```

Comment:
Why adding a onInterim field?
Calls on interim should be like a normal makeContractCall on a pTokensEvmProvider running on the interim chain.